h/static/scripts/annotation-metadata.js

```diff
@@ -70,6 +70,36 @@ function isNew(annotation) {
   return !annotation.id;
 }
 
+function isTypePageNote(annotation) {
+  if (annotation && !annotation.deleted) {
+    if (isTypeReply(annotation)) {
+      return false;
+    }
+    if (annotation.target && (annotation.target.length === 0 || !annotation.target[0].selector)) {
+      return true;
+    }
+  }
+  return false;
+}
+
+function isTypeAnnotation(annotation) {
+  if (annotation && !annotation.deleted) {
+    if (annotation.target && annotation.target.length > 0 && annotation.target[0].selector) {
+      return true;
+    };
+  }
+  return false;
+}
+
+function isTypeReply(annotation) {
```

Comment:
There is already an `isReply` function in this file. Any reason not to use that?